website/docs/migration/from_state_notifier/family_and_dispose/family_and_dispose.dart

```diff
@@ -0,0 +1,26 @@
+// ignore_for_file: unnecessary_this
+
+import 'dart:math';
+
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+import '../../utils.dart';
+
+part 'family_and_dispose.g.dart';
+
+/* SNIPPET START */
+@riverpod
+class BugsEncounteredNotifier extends _$BugsEncounteredNotifier {
+  @override
+  FutureOr<int> build(String featureId) {
+    return 99;
+  }
+
+  Future<void> fix(int amount) async {
+    state = await AsyncValue.guard(() async {
```

Comment:
This does not follow the "Side-effect" page recommendations.

Don't use guard and use the same syntax as the side-effect page instead :)